scalyr_agent/builtin_monitors/syslog_monitor.py

```diff
@@ -128,11 +128,20 @@
     "Note: RFC 5425 (syslog over TCP/TLS) says syslog receivers MUST be able to support messages at least 2048 bytes long, and recommends they SHOULD "
     "support messages up to 8192 bytes long.",
     default=8192,
-    min_value=2048,
     max_value=65536 * 1024,
     convert_to=int,
 )
 
+define_config_option(
+    __monitor__,
+    "tcp_unlimited_buffer_size",
```

Comment:
Yeah, this name might not be the best because TCP/IP buffer size probably means something very specific to folks.

How about these as alternatives:
1. `limit_line_size_to_tcp_buffer`
2. `allow_message_to_exceed_tcp_buffer`
3. `message_size_can_exceed_tcp_buffer`
4. `lines_can_span_tcp_buffers`

Ok, I'm just rattling off combinations at this point.  See if any of those suggestions work for you.